_layouts/post.html

```diff
@@ -46,6 +46,9 @@ <h1 itemprop="name headline" class="u-textCenter u-textWeight-700 u-text-r-xxl u
             {% if page.subtitle %}
```

Comment:
Se `subtitle` non è più presente, questo `if` non è necessario; in compenso ora si è perso l'`if` nel caso `page.description` non sia presente.